### DISCUSSION TITLE
Preventing HDD to go sleep?

### DISCUSSION THREAD

**Discussion by Bobika2002** [Rem0o/FanControl.Releases#1185] [Q&A] [answered]
Does fan control prevents HDD from sleep? Im not using any HDD temp in fan control but it seems like hdd´s wont go sleep because its still monitoring their temp is that possible? Thanks

**Comment by Rem0o** [comment, marked as answer]
If you got HDD enabled in the sensor settings, yes. Disable it if you don't need it.

**Comment by Bobika2002** [comment]
Thanks, I feel very stupid now. I didn´t thought of disabling it. Thanks again and sorry...
